.github/workflows/rust.yml

```diff
@@ -13,6 +13,7 @@ on:
 env:
   CARGO_TERM_COLOR: always
   RUST_VERSION: 1.67.1
+  HEAD_REF: {{ github.head_ref }}
```

Comment:
I think the syntax for using these variables in github actions is to lead with a dollar sign, so `HEAD_REF: ${{ github.head_ref }}`

Reply:
Takes a village, or a Naomi.